admin/tests/integration/components/campaigns/update_test.js

```diff
@@ -163,4 +168,61 @@ module('Integration | Component | Campaigns | Update', function (hooks) {
       assert.dom(screen.queryByRole('checkbox', { name: 'Envoi multiple' })).doesNotExist();
     });
   });
+
+  module('is for absolute novice choice', function () {
+    module('Campaign is Type Assessment', function (hooks) {
+      hooks.beforeEach(function () {
+        this.campaign.isTypeAssessment = true;
+      });
+
+      test('should not display choice if hasAccessToCampaignIsForAbsoluteNoviceEditionScope is false', async function (assert) {
+        // given
+        class AccessControlStub extends Service {
+          hasAccessToCampaignIsForAbsoluteNoviceEditionScope = false;
+        }
+
+        this.owner.register('service:access-control', AccessControlStub);
+
+        // when
+        const screen = await render(hbs`<Campaigns::update @campaign={{this.campaign}} @onExit={{this.onExit}} />`);
+
+        assert.notOk(
+          screen.queryByRole('radiogroup', { name: 'Voulez-vous passer cette campagne en isForAbsoluteNovice' }),
+        );
+      });
+
+      test('should display choice if hasAccessToCampaignIsForAbsoluteNoviceEditionScope is true', async function (assert) {
+        // given
+        class AccessControlStub extends Service {
+          hasAccessToCampaignIsForAbsoluteNoviceEditionScope = true;
+        }
+
+        this.owner.register('service:access-control', AccessControlStub);
+
+        // when
+        const screen = await render(hbs`<Campaigns::update @campaign={{this.campaign}} @onExit={{this.onExit}} />`);
+
+        assert.ok(screen.getByRole('radiogroup', { name: 'Voulez-vous passer cette campagne en isForAbsoluteNovice' }));
+      });
+    });
+  });
+
+  module('Campaign is Type Profiles Collection', function () {
+    test('should display choice if hasAccessToCampaignIsForAbsoluteNoviceEditionScope is true', async function (assert) {
```

Comment:
**suggestion**: should not display choice even if hasAccessToCampaignIsForAbsoluteNoviceEditionScope is true